docs/AboutUs.md

```diff
@@ -14,8 +14,8 @@ We are a team based in the [School of Computing, National University of Singapor
 [[github](https://github.com/russelltankaimin)]
 [[portfolio](team/russelltankaimin.md)]
 
-* Role: Developer
-* Responsibilities: TBD
+* Role: Developer, Team Lead
+* Responsibilities: `view` command, `TimingRecommender`, Deliverables, Deadlines, Gradle issues
```

Comment:
`view` should this be `View` instead? There is an occurrence of the capitalised version below.